### ISSUE TITLE
Color Contrast Testing

### ISSUE THREAD

**Issue by brettfiedler** [phetsims/membrane-transport#353]
After issues #257 and #227 are checked during phet design meeting, I believe we are good for color contrast testing if QA has time! 

If the design doc is being used to catalog this, it can be found here: https://docs.google.com/document/d/1s9PeHpgyg6LgbD-YDoeIQsHinqLpe96YIcwJiPuUfJk/edit?usp=sharing

**Comment by brettfiedler** [comment]
#257 and #227 are complete. We're clear for contrast testing if QA has some room!

**Comment by brettfiedler** [comment]
Everything should be updated to cover contrast concerns now. No exceptions for anything are required.
